include/at_cmd_parser/at_cmd_parser.h

```diff
@@ -63,15 +78,28 @@ int at_parser_max_params_from_str(char *at_params_str,
  *
  * @param at_params_str AT parameters as a null-terminated string. Can be
  *                      numeric or string parameters.
+ *
+ * @next_param_str      In the case a string contains multiple notifications,
+ *                      the parser will stop parsing when it is done parsing
+ *                      the first notification, and return the remainder of
+ *                      the string in this pointer. The return code will be
+ *                      EAGAIN. If multinotification is not used, this
+ *                      pointer can be set to NULL.
+ *
  * @param list          Pointer to an initialized list where parameters
  *                      are stored. Must not be NULL.
  *
  * @retval 0 If the operation was successful.
  *           Otherwise, a (negative) error code is returned.
```

Comment:
I think it would help if you could document/list all error codes. It may not be obvious what the error `E2BIG` means for instance.
Same comment for the `at_parser_params_from_str` function.

Reply:
OK

Reply:
You should remove the line "Otherwise, a (negative) error code is returned.".